doc/tc-and-l3-offload.md

```diff
@@ -131,6 +131,38 @@ Routes offloaded to the device are labeled with `offload` in the ip route listin
 192.168.3.0/24 dev tsn2 proto kernel scope link src 192.168.3.1 offload
 ```
 
+### VLAN L3 offload
+The procedure is the same as in testing regular L3 offload but done to VLAN devices that have the same VLAN id.
+Example command for creating VLAN devices setup for testing
+
+#### BOARD
+```
+ifconfig tsn1 up 192.168.2.1
+ip link add link tsn0 name tsn0.10 type vlan id 10
+ifconfig tsn0.10 up 192.168.10.1
+ip link add link tsn1 name tsn1.10 type vlan id 10
+ifconfig tsn1.10 up 192.168.21.1
+```
+#### HOST TSN0 namespace
+```
+ip link add link eth-tsn0 name eth-tsn0.10 type vlan id 10
+ifconfig eth-tsn0.10 up 192.168.10.100
+sudo ip route add 192.168.21.0/24 via 192.168.10.1
+```
+#### HOST TSN1 namespace
```

Comment:
Please add "or TSN1 connected PC (in case of 2 PC setup)" to be in common with regular FIB instruction